### DISCUSSION TITLE
Welcome to ChatGPT-Telegram-Workers Discussions!

### DISCUSSION THREAD

**Discussion by tbxark** [TBXark/ChatGPT-Telegram-Workers#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wei-haoshuai** [comment]
请问这个有再细致的操作嘛，见谅

**Comment by xiangyingchang** [comment]
I came here specifically to thank the host. I have no understanding of coding, but I've successfully deployed my own chatbot. 

I'm already using AzureOpenAI, which is very convenient. I also have two small suggestions, although they don't affect the core functionality. 

Firstly, it would be great if we could choose the model because it seems we can currently only set one. 

Secondly, it would be helpful to support selecting multiple roles.

 Thank you again, and I'll provide more feedback if I have other suggestions in the future.

**Comment by tianxing8899** [comment]
![image](https://github.com/user-attachments/assets/c92d92d9-aade-4b1e-adc0-e8aa975f9972)
请问遇到这样的怎么处理

**Reply by tianxing8899** [reply]
那个OpenAI API  需要付钱。OpenAI已经没有免费额度。自己有无法绑卡充值。问题大概就在这里
